pkg/detectors/rentman/rentman.go

```diff
@@ -0,0 +1,75 @@
+package rentman
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"rentman"}) + `\b([a-zA-Z0-9._]{300,500})\b`)
```

Comment:
suggestion: this regex can be made a bit stricter by adding `ey` as it always starts with `\b(ey[a-zA-Z0-9]{34}.ey[a-zA-Z0-9._-]{250,300})` and they are all split into two section by `.` separator so we can use that to our advantage.